test/helpers.js

```diff
@@ -0,0 +1,5 @@
+export function createTimeStamp({date}) {
+  const isoString = date.toISOString();
+  // remove the milliseconds from the iso time stamp
```

Comment:
Why remove the milliseconds?

Reply:
The library should of course handle both. The fixtures for the test suite could use either (or both). Implementers should be ready for any valid `xsd:datetime` value. I think that's what you achieved here. Not sure removing the milliseconds is necessary...but it's not necessarily harmful either (in the context of fixture data).